### PR TITLE
Fix incorrect vulnerable manifest path check

### LINKED ISSUE #185: Metadata fields `alert-state`, `ghsa-id` & `cvss` are never populated when the manifest file is at the root
If you look at https://github.com/SalimBensiali/le-blanc-jewellery/runs/5561937034?check_suite_focus=true, you would expect to see the relevant vulnerability alert metadata, but I am always getting the default data instead, ie:

```
outputs.alert-state: ''
outputs.ghsa-id: ''
outputs.cvss: 0
```

- I am using a PAT with enough permissions for successful GraphQL API calls
- I have double checked that the PR is indeed fixing a security vulnerability https://github.com/SalimBensiali/le-blanc-jewellery/pull/215

image

### PR DESCRIPTION
**Pull request by SalimBensiali**
Closes #185

### CODE REVIEW

**Comment by mwaddell** [comment]
@SalimBensiali I don't believe that this PR addresses the issue you reported:

image

**Comment by SalimBensiali** [comment]
This https://github.com/dependabot/fetch-metadata/pull/186/commits/b31caa427946df808241624a181d6966a88fa5c8 does expose a bug in how `getAlert` works. There was no coverage for when the manifest is at the root of the repo

File: src/dependabot/verified_commits.ts [hunk 1]
```diff
@@ -78,7 +78,7 @@ export async function getAlert (name: string, version: string, directory: string
 
   const nodes = alerts?.repository?.vulnerabilityAlerts?.nodes
   const found = nodes.find(a => (version === '' || a.vulnerableRequirements === `= ${version}`) &&
-      trimSlashes(a.vulnerableManifestPath) === `${trimSlashes(directory)}/${a.vulnerableManifestFilename}` &&
```
Comment (SalimBensiali) [inline src/dependabot/verified_commits.ts:81]: @mwaddell without the propsed change, when `package.json` is at root, you get the following:
- `a.vulnerableManifestPath` equals `/package.json` which makex `trimSlashes(a.vulnerableManifestPath)` resolve to `package.json`
- `` `${trimSlashes(directory)}/${a.vulnerableManifestFilename}` `` is equivalent to `` `${trimSlashes('/')}/package.json` `` which yields `/package.json`

And so basically the slash trimmed paths never match, ending up making `getAlert` never return a matching security alert

**Comment by SalimBensiali** [comment]
@mwaddell I run the dry-run command to verify on my end

File: src/dependabot/verified_commits.ts (hunk 1 above)
Comment (mwaddell) [reply src/dependabot/verified_commits.ts:81]: I understand now - thank you for the clarification

**Review by mwaddell — DISMISSED** [review]
@brrygrdn please merge

**Review by brrygrdn — DISMISSED** [review]

**Comment by brrygrdn** [comment]
Thanks for this @SalimBensiali - it looks like the `dist/` folder may be out of sync - can you run `npm run build` and check in any generated changes?

**Comment by SalimBensiali** [comment]
Done ✅

**Review by brrygrdn — APPROVED** [review]
🚀

**Pull request merged**